File: 100 days of leetcode/Day9.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        int max = candies[0];
        int len = candies.length;
        for(int i = 0; i < len; i++){
            if(max < candies[i]){
                max = candies[i];
            }
        }
        //Finding the maximum candies given to a child
        List<Boolean> result = new ArrayList<>(len);
        for(int i = 0; i < len; i++){
            if(candies[i] + extraCandies >= max){
                result.add(true);
            }
            else{
                result.add(false);
            }
        }
        return result;
    }
}

File: 100 days of leetcode/Day9.py
class Solution(object):
    def kidsWithCandies(self, candies, extraCandies):
        """
        :type candies: List[int]
        :type extraCandies: int
        :rtype: List[bool]
        """

        max_candies = max(candies)
        result = []
        for candy_value in candies:
            if candy_value + extraCandies >= max_candies:
                result.append(True)
            else:
                result.append(False)
        return result
